plugins/coverbrowser.koplugin/main.lua

```diff
@@ -509,30 +509,16 @@ function CoverBrowser.initGrid(menu, display_mode)
     menu.display_mode_type = display_mode and display_mode:gsub("_.*", "") -- "mosaic" or "list"
 end
 
-function CoverBrowser:refreshFileManagerInstance(cleanup, post_init)
+function CoverBrowser:refreshFileManagerInstance()
     local fc = self.ui.file_chooser
     if fc then
-        if cleanup then -- clean instance properties we may have set
```

Comment:
Trusting you on this simplification (of my code, that I don't want to re-read/re-understand :))